src/Components/Facility/ConsultationDetails/index.tsx

```diff
@@ -126,6 +128,20 @@ export const ConsultationDetails = (props: any) => {
             data.symptoms_text = symptoms.join(", ");
           }
           setConsultationData(data);
+          const assetRes = await dispatch(
+            listAssetBeds({
+              bed: data?.current_bed?.bed_object?.id,
+            })
+          );
+          const isCameraAttachedRes =
+            assetRes.data.results.filter(
+              (asset: { asset_object: { meta: { asset_type: string } } }) => {
+                return asset?.asset_object?.meta?.asset_type === "CAMERA"
+                  ? true
+                  : false;
```

Comment:
You are effectively doing:
```
if (true) {
  return true;
} else {
  return false;
}
```

When you can simply just use the condition itself.

```suggestion
                return asset?.asset_object?.meta?.asset_type === "CAMERA";
```

Reply:
I just copied and pasted the code that retrieves the camera asset present in Feed.tsx file. Anyway, I will change it.

Reply:
Copy-pasting can sometimes inadvertently propagate legacy patterns. It's always good to revisit and refine the code we inherit. 😄 Looking forward to the updated version! You can also refactor the existing code you copied from.

Reply:
> Copy-pasting can sometimes inadvertently propagate legacy patterns. It's always good to revisit and refine the code we inherit. 😄 Looking forward to the updated version! You can also refactor the existing code you copied from.

Yeah